src/test/java/org/broadinstitute/hellbender/utils/FisherExactTestUnitTest.java

```diff
@@ -26,6 +26,8 @@
         tests.add(new Object[]{1, 2, 3, 4,                     1.0});
         tests.add(new Object[]{0, 0, 100000, 100000,           1.0});
         tests.add(new Object[]{100000,100000,100000,0,         0.0});             //below R's or Java's precision
+        tests.add(new Object[]{12, 4, 26, 7,                   1.0});
```

Comment:
One of them does (the second one), I added both because the reversed one fails in GATK3.